master/usage/configuration/mtu.md

```diff
@@ -82,3 +82,12 @@ calicoctl config set --raw=felix IpInIpMtu 1480
 
 > **Note:** Setting the `IpInIpMtu` config option will result in an immediate
 update the tunnel interface MTU on all of the active nodes in your cluster.
+
+## Kubernetes
+
+If using Kubernetes, the changes listed above should be made to your manifest
```

Comment:
I don't follow what this first sentence is trying to say. A small nit is that it doesn't state which manifest it is talking about (Calico's hosted installation manifests). But a bigger nit - It makes it sounds like I have to do something special to ensure they are pemanent, applied to new nodes, and are not cleared on reboot. But I'm guessing you're just describing what happens when the manifests are applied?

I agree with Casey. Instead of describing which lines of a manifest to edit, we should add these to the manifests explicitly. Then this paragraph just becomes: "The above settings can be specified in the hosted install manifests"